.circleci/config.yml

```diff
@@ -42,7 +42,38 @@ jobs:
           name: Run Tests
           command: yarn test --coverage
 
-  deploy:
+      - run:
+          name: Build
+          command: yarn build
+
+      - run:
+          name: Remove Mock Media Files
+          command: rm -rf build/music
+
+      - run:
+          name: Compress Build Artifacts
+          command: tar czvf forte-web.tar.gz -C build .
+
+      - persist_to_workspace:
+          root: .
+          paths:
+            - ./forte-web.tar.gz
+
+  deploy_artifacts:
+    docker:
+      - image: garland/aws-cli-docker:1.15.47
+
+    steps:
+      - attach_workspace:
+          at: .
+
+      - run:
+          name: Upload to S3
+          command: |
+            FILE_NAME=$([ -z "$CIRCLE_TAG" ] && echo "$CIRCLE_BRANCH" || echo "$CIRCLE_TAG")
+            aws s3 cp ./forte-web.tar.gz "s3://forte-web-artifacts/${FILE_NAME}.tar.gz"
```

Comment:
Do you really want the final segment of the branch/tag name to be the file name? Or you could have a static file name like `forte-web.tar.gz` and use the branch/tag for just the folder.

Reply:
That sounds better. I'll change the name to `s3://${FILE_NAME}/forte-web.tar.gz` ?